### PR TITLE
making fraud-new-isoscape trigger fraud-update sample

### PR DESCRIPTION
**Pull request by kazob1998**
tested on river-sky, this PR addresses https://github.com/tnc-br/bugbash/issues/70 
This PR:
- deletes the old-version folder
- make fraud-new-isoscape minimal to trigger update-sample

### CODE REVIEW

File: fraud-detection-new-isoscape/main.py [hunk 1]
```diff
@@ -6,91 +6,58 @@
...
-        credentials = ee.ServiceAccountCredentials(
-            service_account, 'key.json')
-        ee.Initialize(credentials)
-        print('initialized with key')
-
-    # if in local env use the local user credential
-    else:
-        ee.Initialize()
+    # Authenticate to Earth Engine using service account creds
+    credentials = compute_engine.Credentials(scopes=['https://www.googleapis.com/auth/earthengine'])
```
Comment (benwulfe) [inline fraud-detection-new-isoscape/main.py:25]: do we need to set up authorization/perms in the PRD project?  do we need to grant EE perms to the trigger service account?

File: fraud-detection-update-sample/tests/test_main.py [hunk 2]
```diff
@@ -7,10 +7,13 @@
...
 sys.path.append(parent_dir)
 from main import *
+# Get function environment variable for GCP project ID to use for accessing Earth Engine.
+GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "river-sky-386919")
+
 
 class TestMain(unittest.TestCase):
     def test_parse_path_parts(self):
-        firestore_payload_name = "projects/river-sky-386919/databases/(default)/documents/untrusted_samples/fd1239d5488215a23440"
+        firestore_payload_name = "projects/{GCP_PROJECT_ID}/databases/(default)/documents/untrusted_samples/fd1239d5488215a23440"
```
Comment (benwulfe) [inline fraud-detection-update-sample/tests/test_main.py:16]: we need better unit tests yes?  isnt this just testing parse_path_parts?

**Review by benwulfe — CHANGES_REQUESTED** [review]

File: fraud-detection-new-isoscape/main.py (hunk 1 above)
Comment (kazob1998) [reply fraud-detection-new-isoscape/main.py:25]: already done

File: fraud-detection-update-sample/tests/test_main.py (hunk 2 above)
Comment (kazob1998) [reply fraud-detection-update-sample/tests/test_main.py:16]: yes .. @jjcastro or @VimWizardVersace  wrote this down

**Review by benwulfe — CHANGES_REQUESTED** [review]
can you list the commands you ran to test?  so others can do the same?

**Comment by kazob1998** [comment]
This was tested by running the function manually on river-sky GCP and update sample was triggered as expected

**Comment by benwulfe** [comment]
> This was tested by running the function manually on river-sky GCP and update sample was triggered as expected

I was looking more for specific commands so someone unfamiliar can run it:

> fraud-new-isoscape.py > out.txt
> cat out.txt | grep expected

output:  expected result here!

**Comment by kazob1998** [comment]
From the cloud shell:
run: curl -m 3610 -X POST https://southamerica-east1-river-sky-386919.cloudfunctions.net/fraud-detection-new-isoscape-sc \
> -H "Authorization: bearer $(gcloud auth print-identity-token)" \
> -H "Content-Type: application/json" \
> -d '{  "name": "Hello World" }'

- output should be:   "Fraud detection on new isoscape executed"

**Pull request merged**